plugins/keepkey/keepkey.py

```diff
@@ -86,40 +87,45 @@ def __init__(self, parent, config, name):
             import keepkeylib.ckd_public
             import keepkeylib.transport_hid
             import keepkeylib.transport_webusb
+            from electroncash_plugins.hw_wallet.utils.libusb import USBContext
```

Comment:
Hey the rest of this file uses relative imports (eg `..hw_wallet`).  I am not clear on the caveats of each -- why did you opt to do it this way?

Reply:
No reason at all, should I change it to relative?

Reply:
Yeah no idea what the caveats are.. figured keep it same same.  Thanks. 👍

Reply:
Changed to relative.